src/plugins/ResizeReorder/ReorderHandle.js

```diff
@@ -0,0 +1,241 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReorderHandle
+ * @typechecks
+ */
+
+import React from 'react';
+import cx from 'cx';
+import PropTypes from 'prop-types';
+import DOMMouseMoveTracker from 'DOMMouseMoveTracker';
+import { getState } from 'FixedDataTableStore';
+
+const DRAG_SCROLL_SPEED = 10;
+const DRAG_SCROLL_BUFFER = 100;
+
+class ReorderHandle extends React.Component {
+
+  static propTypes = {
+    height: PropTypes.number,
+    touchEnabled: PropTypes.bool,
+    isRTL: PropTypes.bool,
+    scrollX: PropTypes.number,
+    maxScrollX: PropTypes.number,
+    left: PropTypes.number,
+    availableScrollWidth: PropTypes.number,
+    isFixed: PropTypes.bool,
+    scrollToX: PropTypes.func,
+    onColumnReorderEndCallback: PropTypes.func,
+    getColumnGroupWidth: PropTypes.func,
+    columnKey: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.number
+    ]),
+  };
+
+  /**
+   * Instance of DOMMouseMoveTracker to capture mouse events
+   * @type {DOMMouseMoveTracker}
+   */
+  mouseMoveTracker = null;
+
+  /**
+   * Displacement of reorder cell while reordering
+   * @type {number}
+   */
+  distance = 0;
```

Comment:
why do we need to track this? Isn't this already passed by the parent?

Reply:
This is distance moved with cursor..Parent stores distance wrt scrollbar

Reply:
Fixed columns can wither be on left or right and hence scrollbar can't be moved when we try to reorder them. This check skips updating scrollbar for fixed columns.